src/routes/article.js

```diff
@@ -0,0 +1,7 @@
+const { Router } = require('express');
```

Comment:
Could you explain me a little about router, i have an idea how it works, but haven't use it.

Reply:
You can refer to this link for further and more detailed information: http://expressjs.com/en/guide/routing.html